### PR TITLE
Player creation and simple room page

### PR DESCRIPTION
**Pull request by DanielMuckelbauer**
Animation

### CODE REVIEW

File: src/app/home/room-creation-modal/room-creation-modal.component.ts [hunk 1]
```diff
@@ -19,12 +20,16 @@ export class RoomCreationModalComponent {
...
+    private router: Router,
     private formBuilder: FormBuilder) {}
 
   cancel(): void {
     this.dialogRef.close();
   }
 
-  create(): void { }
+  create(): void {
+    this.router.navigate(['room', this.formGroup.value.name]).then();
```
Comment (Hakem22) [inline src/app/home/room-creation-modal/room-creation-modal.component.ts:32]: Maybe constant for this as well?

**Comment by Hakem22** [comment]
You didn't handle the copy link I see :)

File: src/app/home/room-creation-modal/room-creation-modal.component.ts (hunk 1 above)
Comment (DanielMuckelbauer) [reply src/app/home/room-creation-modal/room-creation-modal.component.ts]: Good idea!

File: src/app/app-routing.module.ts [hunk 2]
```diff
@@ -1,7 +1,12 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import { HomeComponent } from './home/home.component';
+import { RoomComponent } from './room/room.component';
 
-const routes: Routes = [];
+const routes: Routes = [
+  { path: '', component: HomeComponent },
+  { path: 'room/:id', component: RoomComponent,  },
```
[the remark this answers is not in the document]
Comment (DanielMuckelbauer) [reply src/app/app-routing.module.ts]: Meh

File: src/app/room/room.component.ts [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    const serializedPlayer = localStorage.getItem(constants.playerKey);
+    if (serializedPlayer) {
+      this.player = JSON.parse(serializedPlayer);
+    } else {
+      this.createPlayer();
+    }
+  }
+
+  // only here so we don't need to do it manually, can be removed when we have a real deletion mechanism
+  deletePlayerInfo(): void {
```
[the remark this answers is not in the document]
Comment (DanielMuckelbauer) [reply src/app/room/room.component.ts]: That is a very good point. workaround to feature!

**Review by Hakem22 — APPROVED** [review]

**Pull request merged**